Peek-swiftUI/Modules/MoviesDetailsView/Views/MovieChip.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  MovieChip.swift
+//  Peek-swiftUI
+//
+//  Created by mazen eldeeb on 27/08/2024.
+//
+
+import SwiftUI
+
+struct MovieChip: View {
+    let text: String
+    
+    var body: some View {
+        Text(text)
+            .foregroundStyle(.white)
+            .padding(.horizontal, 14)
+            .padding(.vertical, 6)
+            .background(.lightPurple)
+            .clipShape(RoundedRectangle(cornerRadius: .infinity))
```

Comment:
set line limi to one and use a small font

Reply:
done

Reply:
use a smaller font. you can can use custom system font and give it the size` .font(.system(size: <#T##CGFloat#>))`

Reply:
done